pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -22,7 +24,19 @@ class Range(Module):
   is_original_torch_module = False
 
   def create_returnn_layer_dict(self, limit, start, delta, dtype, sparse=False) -> Dict[str, Any]:
-    return {"class": "range", "limit": limit, "start": start, "delta": delta, "dtype": dtype, "sparse": sparse}
+    if isinstance(limit, SizeValue):
```

Comment:
I think this only should be done for dynamic dims. Specifically:
```suggestion
    if isinstance(limit, SizeValue) and limit.dim_tag.dimension is None:
```

Reply:
Yes, this fixes `test_arange` and the issue mentioned in your comment above

>But T|'Range_input_len'[] looks wrong. This is marked as a dynamic dim. But it should be static (right)?